doc/src/devdocs/llvm.md

```diff
@@ -154,11 +154,15 @@ using:
 fun, T = +, Tuple{Int,Int} # Substitute your function of interest here
 optimize = false
 open("plus.ll", "w") do file
-    println(file, InteractiveUtils._dump_function(fun, T, false, false, false, true, :att, optimize, :default, false))
+    println(file, InteractiveUtils._dump_function(fun, T, false, false, true, true, :att, optimize, :default, false))
```

Comment:
I am unsure why we don't just do

``code_llvm(io, fun, T, raw=true, dump_module=true)` instead of using this lower level function.

Reply:
probably because `code_llvm` dumps to your terminal rather than to a string so redirecting it to a file is harder.

Reply:
What Valentin is saying is that you can do
```julia
open("/tmp/exp.ll", "w") do io
    code_llvm(io, exp, (Float64,); raw=true, dump_module=true)
end
```
because `code_llvm` does take an `IO` as input

Reply:
Concretely, his suggestion was
```suggestion
    code_llvm(file, fun, T; raw=true, dump_module=true, optimize)
```